lib/cartopy/io/img_nest.py

```diff
@@ -142,14 +142,21 @@ def world_files(fname):
                 result = ['{}.{}'.format(froot, ext) for ext in fext_types]
 
         def _convert_basename(name):
-            dirname, basename = os.path.dirname(name), os.path.basename(name)
+            if '\\' in name:
```

Comment:
This is only here so that testing Windows paths works on Linux, as `os.path.split` only works on native paths. I'm not sure if this is the best idea, since Windows accepts both forward and backward slashes. Alternatively, we can just test Linux paths on Linux and Windows paths on Windows. I don't really know what this code is aiming to do, so I don't know which is the better way forward.

Reply:
Yeah, let's just do Windows stuff on windows, and Linux stuff on Linux in this case. Since the filesystem isn't case sensitive on Windows you could argue that it should actually produce different results Linux.